app/components/node/PoSSize.tsx

```diff
@@ -142,6 +152,28 @@ const PoSSize = ({
     setNumUnit(commitments[index].numUnits);
   };
 
+  const validate = (value: number) => {
```

Comment:
I believe the proper name for this function is `constrainPosMaxFileSize`, which can be more expressive by reusing the `constrain` utility function:

```suggestion
  const constrainPosMaxFileSize = (value: number) => constrain(
    DEFAULT_POS_MAX_FILE_SIZE, 
    DEFAULT_POS_MAX_FILE_SIZE_LIMIT,
    Number.isNaN(value) ? 0 : value
  );
```

At least, I'd expect to have a different output from the validate function (like `Either<Error, number>`) :)